routes/index.js

```diff
@@ -246,6 +233,8 @@ module.exports = function (passport) {
     dbAPI.deleteUser(DeletedBadge).then(function (result) {
       console.log("user is deleted successfully");
     });
+    req.flash('success', 'This user is deleted from the lab');
```

Comment:
Actually I forgot this as well. Could you please change this to "The user has been successfully deleted from the system"